templates/taobao-creative-app/app.ejs

```diff
@@ -0,0 +1,51 @@
+let onShowCB;
+let onHideCB;
+App({
+  onLaunch: function onLaunch(options) {
+    console.info('App onLaunched');
+
+    $global.__cocosCallback = function () {
+      require('./<%= polyfillsBundleFile %>');
+      require('./<%= systemJsBundleFile %>');
+      require('./load-module.js');
+      const importMap = require('./<%= importMapFile %>').default;
+      let System = $global.System;
+      System.warmup({
+        importMap,
+        importMapUrl: 'src/import-map.js',
+      });
+      System.import('./web-adapter.js')
+      .then(() => {
+        return System.import('cc');
+      })
+      .then((cc) => {
+        return System.import('./engine-adapter.js')
+        .then(() => {
+          return System.import('<%= applicationJs %>')
+          .then(({ Application }) => {
+            const application = new Application();
+            application.init(cc);
+            return application.start();
+          });
+        });
+      })
+      .catch((err) => {
+        console.error(err);
+      });    
+    };
+
+    my.onShow = function (cb) {
+      onShowCB = cb;
```

Comment:
why need this?

Reply:
Maybe move to unify.js ? And in this file, only invoke my.dispatchEvent('show');